diff --git a/node_modules/react-native-youtube-iframe/src/YoutubeIframe.js b/node_modules/react-native-youtube-iframe/src/YoutubeIframe.js
index 4bbf487..fd940e9 100644
--- a/node_modules/react-native-youtube-iframe/src/YoutubeIframe.js
+++ b/node_modules/react-native-youtube-iframe/src/YoutubeIframe.js
@@ -121,6 +121,12 @@ const YoutubeIframe = (props, ref) => {
           PLAYER_FUNCTIONS.seekToScript(seconds, allowSeekAhead),
         );
       },
+      pauseVideo: () => {
+        webViewRef.current.injectJavaScript(PLAYER_FUNCTIONS.pauseVideo);
+      },
+      playVideo: () => {
+        webViewRef.current.injectJavaScript(PLAYER_FUNCTIONS.playVideo);
+      },
     }),
     [],
   );
